spec/models/users_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'John', post_counter: 30, photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.') }
+
+  before { subject.save }
+
+  it 'name should be present' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'post_counter should be present' do
+    subject.post_counter = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'post_counter should be integer' do
+    subject.post_counter = 'a'
+    expect(subject).to_not be_valid
+  end
+
+  it 'post_counter should be greater than or equal to zero' do
+    subject.post_counter = -1
+    expect(subject).to_not be_valid
+  end
```

Comment:
- [ ] As stated by the guidelines, kindly add unit specs for all of your models' methods and validations.... Please add test for the user model method, ...` most_recent_posts` please ensure your test are working well. Nice work.

Reply:
* Good job working on the requested changes but i noticed some errors while running your tests below is a screenshot of the said error Kindly fix them. 
![image](https://github.com/ishimwepati/Blog-Rails/assets/52497246/de1ea8fc-ea1d-413d-aad3-62d006fbf327)